terraform-infra-approvals/ccd-elastic-search.json

```diff
@@ -0,0 +1,12 @@
+{
+  "resources": [],
+  "module_calls": [
+    {"source":  "git@github.com:hmcts/cnp-module-adf?ref=master"},
+    {"source":  "git@github.com:hmcts/cnp-module-waf?ref=ccd/CHG0033576"},
+    {"source":  "git@github.com:hmcts/cnp-module-storage-account.git?ref=0.0.1"},
```

Comment:
why not master?

Reply:
I'll tidy these up once I get ES 7.4 deployed in perftest